okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/ConcurrencyLimiters.java

```diff
@@ -208,6 +213,9 @@ public void onDropped() {}
         }
 
         synchronized void processQueue() {
+            // Too many metrics to emit per endpoint, so we record as histogram instead
+            acquireQueue.update(waitingRequests.size());
```

Comment:
so if we end up with a single endpoint that is fast, it'll contribute far more frequently to this histogram which is just gonna skew things here

Reply:
I wonder if you just want to register a gauge which sums all of the queued requests across endpoints and threads